src/test/java/org/folio/dao/DataImportQueueDaoTest.java

```diff
@@ -54,118 +66,170 @@ public void setUp() {
   public void shouldGetQueueItem(TestContext context) {
     // given
     doAnswer((InvocationOnMock invocation) -> {
-      Promise<RowSet<Row>> promise = invocation.getArgument(2);
-      promise.complete(new LocalRowSet(5));
-      return null;
-    }).when(postgresClient)
-      .execute(anyString(), any(Tuple.class), ArgumentMatchers.<Promise<RowSet<Row>>>any());
+        Promise<RowSet<Row>> promise = invocation.getArgument(1);
+        promise.complete(new LocalRowSet(5));
+        return null;
+      })
+      .when(postgresClient)
+      .select(anyString(), ArgumentMatchers.<Promise<RowSet<Row>>>any());
 
     // when
     String query = "size > 1";
-    queueItemDaoImpl.getQueueItems(query,0, 6)
+    queueItemDaoImpl
+      .getQueueItems(query, 0, 6)
       // then
-      .onComplete(context.asyncAssertSuccess(x -> {
-        verify(postgresClient, times(1)).execute(anyString(), any(Tuple.class), ArgumentMatchers.<Promise<RowSet<Row>>>any());
-        verifyNoMoreInteractions(postgresClient);
-      }));
+      .onComplete(
+        context.asyncAssertSuccess(x -> {
+          verify(postgresClient, times(1))
+            .select(
+              startsWith("SELECT * FROM data_import_global.queue_items"),
```

Comment:
For extra confidence, you can use matchers like `startsWith` or `contains` to ensure the "run" SQL queries look the way they should